examples/rustls-mtls/certs/generate_certs.sh

```diff
@@ -0,0 +1,24 @@
+# immediately bail if any command fails
+set -e
+
+echo "generating CA private key and certificate"
+openssl req -nodes -new -x509 -keyout ca-key.pem -out ca-cert.pem -days 65536 -config config/ca.cnf
+
+# use rsa because it lets us have one fewer config file
```

Comment:
what additional config file is needed for EC?

Reply:
> All other algorithms support the -newkey alg:file form, where file may be an algorithm parameter file, created by the genpkey -genparam command or and X.509 certificate for a key with approriate algorithm.

https://www.openssl.org/docs/man1.0.2/man1/openssl-req.html

Reply:
Ah ok. Looks like you can also use the `-pkeyopt` option too: https://security.stackexchange.com/a/190646